multisense_ros/cfg/multisense.cfg

```diff
@@ -215,8 +215,8 @@ for cfg in sensorConfigList:
         gen.add("detail_disparity_profile", bool_t, 0, "DetailDisparityProfile", False);
         gen.add("high_contrast_profile", bool_t, 0, "HighContrastProfile", False);
         gen.add("show_roi_profile", bool_t, 0, "ShowRoiProfile", False);
-        gen.add("ground_surface_profile", bool_t, 0, "GroundSurfaceProfile", False);
         gen.add("full_res_aux_profile", bool_t, 0, "FullResAuxProfile", False);
+        gen.add("ground_surface_profile", bool_t, 0, "GroundSurfaceProfile", False);
```

Comment:
Note that in a future PR I could hide these options for cameras which don't support this feature.

We could do this by generating additional configs for on-camera algorithms -> then in `reconfigure.cpp` we could query the device info, and alongside the camera type we would get the list of supported on-camera algorithms & then could select the appropriate config.

Reply:
Would it be possible to integrate that "hiding" feature into this PR. We should be able to check what streams are supported with the Device modes and select the appropriate dynamic reconfigure config. 

I am just worried this will lead to a ton of questions from current customers.

Reply:
Ok, for sure. I agree that it's best to avoid confusion for current customers!